src/Chapter2.hs

```diff
@@ -741,7 +766,8 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
+contains _ [] = []
+contains x xs = filter (elem x) xs
```

Comment:
you don't have to check the empty list separately as `filter` works on any lists 🙂 

Also, then you can see that you can apply eta-reduction to this function as well  🙂

Reply:
I see:

`contains x = filter (elem x)` is all that's needed